gprofiler/main.py

```diff
@@ -103,16 +107,28 @@ def _generate_output_files(
         collapsed_data: str,
         local_start_time: datetime.datetime,
         local_end_time: datetime.datetime,
-        flamegraph: bool,
     ) -> None:
         start_ts = get_iso8061_format_time(local_start_time)
         end_ts = get_iso8061_format_time(local_end_time)
         base_filename = os.path.join(self._output_dir, "profile_{}".format(end_ts))
+
         collapsed_path = base_filename + ".col"
         Path(collapsed_path).write_text(collapsed_data)
+
+        # point last_profile.col at the new file; and possibly, delete the previous one.
+        last_profile = os.path.join(self._output_dir, "last_profile.col")
```

Comment:
It always return the same value, can be initialized in `__init__`?

Reply:
Hmm yeah but it's really only private to this function so I see no reason to store it as an attribute (it's not that this "calculation" costs time)

Reply:
Hmm yeah but it's really only private to this function so I see no reason to store it as an attribute (it's not that this "calculation" costs time)